src/screens/ProfileScreen/elements/LinkAndSocialMedia.tsx

```diff
@@ -42,7 +43,8 @@ const LinkAndSocialMedia: React.FC<LinkProps> = ({username}) => {
     <View
       style={{
         borderWidth: 1,
-        borderColor: COLORS.gray,
+        borderColor: COLORS.gray200,
+        backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderColor and backgroundColor have been updated to use different shades of gray. Ensure these color changes align with the overall design system and maintain sufficient contrast for accessibility.
```diff
-        borderColor: COLORS.gray,
+        borderColor: COLORS.gray200,
+        backgroundColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->